File: webview/src/main/java/com/epro/webview/MainActivity.java
package com.epro.webview;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private EditText mEditText;
    private TextView mTextView;
    public static final String WEB_VIEW_EXTRA = "WEB_VIEW_EXTRA";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mEditText = findViewById(R.id.et);
        mTextView = findViewById(R.id.tv);
        mTextView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, WebviewActivity.class);
                intent.putExtra(WEB_VIEW_EXTRA, mEditText.getText().toString().trim());
                startActivity(intent);
            }
        });
    }
}

File: kotlin/src/main/java/com/epro/kotlin/MainActivity.kt
package com.epro.kotlin

import android.os.Bundle
import android.view.View
import android.view.ViewGroup
import android.view.WindowManager
import androidx.appcompat.app.AppCompatActivity
import androidx.constraintlayout.widget.ConstraintLayout
import com.blankj.utilcode.util.LogUtils
import com.blankj.utilcode.util.ScreenUtils
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var lastDoubleClickTime = 0L;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

//        TestRunnableActivity.startTestRunnableActivity()
//        ExecuteOrderActivity.startExecuteOrderActivity()


        tv_hello_world.setOnClickListener(View.OnClickListener {
            LogUtils.e(System.currentTimeMillis(), lastDoubleClickTime, System.currentTimeMillis() - lastDoubleClickTime)
            if (System.currentTimeMillis() - lastDoubleClickTime < 500) {
                //切换全屏
                LogUtils.e("simpleExoPlayerView double click")
                val layoutParams = ConstraintLayout.LayoutParams(tv_hello_world.layoutParams)
                if (!ScreenUtils.isFullScreen(this@MainActivity)) {
                    ScreenUtils.setFullScreen(this@MainActivity)
                    layoutParams.width = ScreenUtils.getScreenWidth()
                    layoutParams.height = ScreenUtils.getScreenHeight()
                } else {
                    ScreenUtils.setNonFullScreen(this@MainActivity)
                    layoutParams.width = ViewGroup.LayoutParams.WRAP_CONTENT
                    layoutParams.height = ViewGroup.LayoutParams.WRAP_CONTENT
                }
                tv_hello_world.setLayoutParams(layoutParams)
                lastDoubleClickTime = 0;
                return@OnClickListener
            }
            lastDoubleClickTime = System.currentTimeMillis()
        })
    }
}
